pulsar-client-cpp/include/pulsar/Schema.h

```diff
@@ -27,6 +27,25 @@
 
 namespace pulsar {
 
+/**
+ *  Encoding types of supported KeyValueSchema for Pulsar messages.
+ */
+enum KeyValueEncodingType
```

Comment:
I recommended `enum class` rather than `enum` for new APIs. With an enum class, you must write `KeyValueEncodingType::INLINE`, but with a normal enum, you can write `KeyValueEncodingType::INLINE` or `INLINE`. See https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum